package.json

```diff
@@ -84,6 +87,7 @@
   },
   "devDependencies": {
     "@babel/core": "7.4.5",
+    "@babel/parser": "7.4.5",
```

Comment:
why do we need this?

Reply:
One of the dependency's declaration files (don't recall which one) requires that.
You can leave it out and you'll see the error if you want to know more about it.